modules/kubernetes-addons/gatekeeper/locals.tf

```
@@ -0,0 +1,29 @@
+locals {
+  name      = "gatekeeper"
+  namespace = "gatekeeper-system"
+
+  default_helm_config = {
+    name             = local.name
+    chart            = local.name
+    repository       = "https://open-policy-agent.github.io/gatekeeper/charts"
+    version          = "3.9.0"
+    namespace        = local.namespace
+    values           = local.default_helm_values
```

this can be simply changed to:

```suggestion
    values           = [
    	<<-EOT
    		clusterName: ${var.addon_context.eks_cluster_id}
    	EOT
    ]
```

which means you can get rid of the `default_helm_values` on line 21 and the `values.yaml` file

---

Fixed